src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1235,6 +1236,46 @@ typedef struct {
                      //!< RTCDataChannel object with the same id as the other peer.
 } RtcDataChannelInit, *PRtcDataChannelInit;
 
+
+/////////////////////////////////////////////////////
+/// Metrics/Stats Related structures
+/////////////////////////////////////////////////////
+
+/**
+ * @brief Collection of ICE related stats
```

Comment:
Could you perhaps include the references for RFC where these are defined?

Reply:
I have added them in Stats.h where the structures are defined. I will also add it here for reference.